docs/tutorials/ua_fix_scenarios.md

```diff
@@ -0,0 +1,365 @@
+you to run that command on your Xenial LXD container.
+
+## Using UA fix
+
+First, let's see what happens on your system when `ua fix` runs. We will choose
+to fix a CVE that does not affect the Xenial container,
+[CVE-2020-15180](https://ubuntu.com/security/CVE-2020-15180). This CVE address security
+issues for the `MariaDB` package, which is not installed on the system. Let's confirm that
+it doesn't affect the system by running this command:
+
+```console
+$ ua fix CVE-2020-15180
+```
+
+You should see an output like this one:
+
+```
+CVE-2020-15180: MariaDB vulnerabilities
+https://ubuntu.com/security/CVE-2020-15180
+No affected source packages are installed.
+✔ CVE-2020-15180 does not affect your system.
+```
+
+Every `ua fix` output will have a similar output structure where we describe the CVE/USN,
+display the affected packages, fix the affected packages and at the end, show if the
+CVE/USN is fully fixed in the machine.
+
+You can better see this on an `ua fix` call that does fix a package. Let's install a package
+on the container that we know are associated with [CVE-2020-25686](https://ubuntu.com/security/CVE-2020-25686).
+You can install that package by running this command:
+
+```console
+$ sudo apt install dnsmasq=2.75-1
+```
+
+Now you can run the following command:
+
+```console
+$ sudo ua fix CVE-2020-25686
+```
+
+You will then see the following output:
+
+```
+CVE-2020-25686: Dnsmasq vulnerabilities
+https://ubuntu.com/security/CVE-2020-25686
+1 affected package is installed: dnsmasq
+(1/1) dnsmasq:
+A fix is available in Ubuntu standard updates.
+{ apt update && apt install --only-upgrade -y dnsmasq }
+✔ CVE-2020-25686 is resolved.
+```
+
+> NOTE: we need to run the command with sudo because we are now installing a package on the system.
+
+Whenever `ua fix` has a package to upgrade, it follows a consistent structure and displays the
+following in order
+
+1. The affected package
+2. The availability of a fix
+3. The location of the fix, if one is available
+4. The command that will fix the issue
+
+Also, in the end of the output is the confirmation that the CVE was fixed by the command.
+You can confirm that fix was successfully applied by running the same `ua fix` command again:
+
+```
+CVE-2020-25686: Dnsmasq vulnerabilities
+https://ubuntu.com/security/CVE-2020-25686
+1 affected package is installed: dnsmasq
+(1/1) dnsmasq:
+A fix is available in Ubuntu standard updates.
+The update is already installed.
+✔ CVE-2020-25686 is resolved.
+```
+
+## CVE/USN without released fix
+
+Some CVE/USNs do not have a fix released yet. When that happens, `ua fix` will let you know
+about this situation. You can confirm that by running the following command:
+
+
+```
+$ ua fix USN-4539-1
+```
+
+You will see the following output:
+
+```
+USN-4539-1: AWL vulnerability
+Found CVEs:
+https://ubuntu.com/security/CVE-2020-11728
+1 affected source package is installed: awl
+(1/1) awl:
+Sorry, no fix is available.
```

Comment:
do we need to instruct the user to install something in order to see this?

Reply:
Good catch again